qiskit_nature/second_q/mappers/qubit_mapper.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, allows_two_qubit_reduction: bool = False):
                 number of qubits in the mapped operator can be reduced accordingly.
         """
         self._allows_two_qubit_reduction = allows_two_qubit_reduction
+        self.times_creation_op = []
```

Comment:
```suggestion
        self.times_creation_op: list[SparsePauliOp] = []
```
I believe this is the type hint you're looking for in all of these cases